README.md

```diff
@@ -115,10 +115,15 @@ cargo build --release --features cuda
     cd mistral.rs
     ```
 
-5) Build
-    ```bash
-    cargo build --release --features cuda
-    ```
+5) Build or install
+    - Build
+        ```bash
+        cargo build --release --features cuda
+        ```
+    - Install with `cargo install` for easy command line usage
+        ```bash
+        cargo build --path mistralrs-server --features cuda
```

Comment:
Shouldn't this be `install` instead of `build`?

Reply:
Yes, you are correct, thanks for catching that! It should be fixed now.